academic-requirements/src/App.js

```diff
@@ -156,6 +158,10 @@ function App() {
     }
   }, [concentration]);
 
+  useEffect(() => {
```

Comment:
I think when we were peer programming I added this useEffect in to sort of test stuff, but it could be removed now.